src/module_tests.c

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
The idea of this file is for any test functions we want temporarily for demo purposes. I'm imagining that after the project has stabilized and the system mostly working, we can get rid of this file in the future in favor of actual OS features.